website-1.x/blog/2021-03-09-releasing-docusaurus-i18n.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Releasing Docusaurus i18n
+author: Sébastien Lorber
+authorTitle: Docusaurus maintainer
+authorURL: https://sebastienlorber.com
+authorImageURL: https://github.com/slorber.png
+authorTwitter: sebastienlorber
+tags: [release, i18n]
+image: /img/blog/2021-03-09-releasing-docusaurus-i18n/social-card.png
+---
+
+Today, we officially release **[Docusaurus 2 i18n](https://v2.docusaurus.io/docs/i18n/introduction)** ([internationalization](https://en.wikipedia.org/wiki/Internationalization_and_localization)), as part of [2.0.0-alpha.71](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-alpha.71).
+
+Docusaurus 2 has now reached **full feature parity with Docusaurus 1**, and a first **beta** is on its way.
+
+**We went the extra mile**, and the new i18n support is **even better than in Docusaurus 1**.
+
+In this post, we will present you the **translation workflow**, explain some **design decisions** and **showcase early adopter sites** that just went live: [Jest](https://jestjs.io/), [Redwood](https://learn.redwoodjs.com/), and [Datagit](https://datagit.ir/).
+
+<!--truncate-->
+
+## Translate your site
+
+You can translate a Docusaurus site in **3 simple steps**:
+
+1. **Configure**: declare the default locale and alternative locales in `docusaurus.config.js`.
+2. **Translate**: put the translation files at the correct file-system location.
+3. **Deploy**: build and deploy your site using a single or multi-domain strategy.
+
+The i18n support is **very flexible** and based on the **file-system**.
+
+The **[i18n tutorial](https://v2.docusaurus.io/docs/i18n/tutorial)** is the best way to get started, and we provide help to use **[Git](https://v2.docusaurus.io/docs/i18n/git)** or **[Crowdin](https://v2.docusaurus.io/docs/i18n/crowdin)**.
+
+## Design decisions
+
+The goals of the Docusaurus i18n system are:
+
+- **Simple**: just put the translated files in the correct file-system location.
+- **Flexible translation workflows**: based on Git (monorepo, forks, or submodules), SaaS software, FTP...
+- **Flexible deployment options**: single, multiple domains, or hybrid.
+- **Modular**: allow plugin authors to provide i18n support.
+- **Low-overhead runtime**: documentation is mostly static and does not require a heavy JS library or polyfills.
+- **Scalable build-times**: allow building and deploying localized sites independently.
+- **Localize assets**: an image of your site might contain text that should be translated.
+- **No coupling**: not forced to use any SaaS, yet the integration is possible.
+- **Easy to use with [Crowdin](https://crowdin.com/)**: multiple Docusaurus 1 sites use Crowdin, and should be able to migrate to v2.
```

Comment:
You use the word "should" here. What would prevent a Docusaurus v1 site from not being able to migrate? Do we have a migration path / information for migrating? I think you can say something like:

> multiple Docusaurus 1 sites use Crowdin, and we continue to support this valuable platform, including migration to v2.